### DISCUSSION TITLE
Single color prints

### DISCUSSION THREAD

**Discussion by ryanlindsley** [tomvandeneede/p2pp#149] [Q&A]
I've been using Palette 3 Pro and P2PP for a little bit. I just now decided to try a single color print. I tried twice, and it resulted in an error (error 149). This occurs right when it should be making its one and only cut so the material can retract and the print continue/finish. 

Has anyone else seen this? Can this be fixed? Does P2PP not support single color prints?

**Comment by xile6** [comment]
Single colored prints should not be sliced with the palette settings.

But sliced normally.
Then thw palette started in multispool mode
With printer.

It will feed the printer filament.
While the printer will read its normal gcode from its own sd/usb drive

**Reply by ryanlindsley** [reply]
Thanks for the reply. I have some followup questions. What I'm trying to do is send the file to the Palette 3 Pro with PrusaSlicer. When doing a multicolor print, I can export with PrusaSlicer to P2PP, and P2PP sends the file to the Palette. 
If I'm doing a single color print, am I still able to send the file to the Palette remotely? Do I still go through P2PP?

If what you're saying is that I basically should not use P2PP for single color prints, that's unfortunate. Going through the Palette with multi-spool mode does not add any value (certainly adds noise!) to my single color prints unless I'm planning on printing large volumes. 

What I was hoping for, was that I can load my Palette with my common colors, and if I wanted to print a single color part, let's say using the color in slot 6, I could slice and send through P2PP just like a multi-color print. That way the setup is consistent no matter what type of print I'm doing.

So far, I've just had to disconnect the Palette and print from the Prusa SD card everytime I want to do single colors. This is exactly what I was hoping to avoid doing in the future.

**Reply by ryanlindsley** [reply]
I found on Facebook and the release notes here the version 8.0.9 supposedly added support (or a fix?) for single color prints. 
If this is supposed to be supported, and I can't get it to work. Help?
